Mathlib/Data/Finsupp/Basic.lean

```diff
@@ -31,9 +28,6 @@ This file is a `noncomputable theory` and uses classical logic throughout.
 
 ## TODO
 
-* This file is currently ~1600 lines long and is quite a miscellany of definitions and lemmas,
-  so it should be divided into smaller pieces.
```

Comment:
Is "quite a miscellany of definitions and lemmas" no longer true?

Reply:
I don't think it's any more miscellaneous than any other Basic.lean file. I assume this comment was made before the long file linter existed and now that it does exist I don't think we need the reminder.

Reply:
Ok, let's leave it deleted as you've done here.